### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by xii-vi**
## Added Video Player mapped to every Video Card.

mobile (20)

### CODE REVIEW

File: src/backend/db/videos.js [hunk 1]
```diff
@@ -2,127 +2,138 @@
...
-    creatorImage: "https://i.ytimg.com/vi/xQsHpqDTL-o/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCgE4VoHtfoKfogy9TL424CSX-uqg"
+    creatorImage: "https://i.ytimg.com/vi/xQsHpqDTL-o/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCgE4VoHtfoKfogy9TL424CSX-uqg",
+    logo:"https://yt3.ggpht.com/ytc/AKedOLQBoBlDevqIuBxrZ82gj_HgCjA_CVJ8yHw-3bRGRw=s176-c-k-c0x00ffffff-no-rj"
   },
   {
-    _id: uuid(),
-    title: "GHAZIABAD",
-    description: "CREDITS:Lyrics/Vocals - Ad Vasudev, Music - Control Gang, Mix/Master - Rakht",
-    creator: "Vasudev",
+    _id: "Fj0S01-kPn8",
```
Comment (ivinayakg) [inline src/backend/db/videos.js:44]: Great that you have shifted to permanent id instead of creating one.

**Review by ivinayakg — APPROVED** [review]
Looks good to merge

File: src/backend/db/videos.js (hunk 1 above)
Comment (xii-vi) [reply src/backend/db/videos.js:44]: Thank You

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

**Review by haarsh24 — COMMENTED** [review]
code looks clean
file structure is good
design looks consistent

**Pull request merged**